src/classes/UTIL_RecordTypes.cls-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>33.0</apiVersion>
+    <apiVersion>37.0</apiVersion>
```

Comment:
I am using v37 of the API for the `Schema.RecordTypeInfo.isMaster()` method.  If you want to avoid mixing version numbers, then hold off on merging this until all the other classes are up to v37.